src/cloud/setup/first-time-setup-import-import.md

```diff
@@ -219,52 +220,65 @@ The {{site.data.var.ee}} encryption key is required as an environment variable i
 
 You copied the key in a [previous step]({{ site.baseurl }}/cloud/setup/first-time-setup-import-prepare.html#encryption-key).
 
-To add your {{site.data.var.ee}} encryption key:
+{:.procedure}
+To add your {{site.data.var.ee}} encryption key using the `CRYPT_KEY` environment variable:
 
-1. If you haven't done so already, SSH to the Cloud environment.
+1. Log in to your Cloud project as an Admin user from the Project Web UI, or from the command line.
+
+1. Set the `CRYPT_KEY` environment variable to the encryption key value that you copied from the remote environment:
+
+   -  From the Project Web UI, select your environment, then select the *Variables* tab to set the `CRYPT_KEY` value. See [Set environment and project variables]({{site.baseurl}}/cloud/project/project-webint-basic.html#project-conf-env-var).
+
+   -  From the command line, use the `project:variable:set` command to add the encryption key to the `CRYPT_KEY` environment variable. See [Working with environment variables]({{ site.baseurl }}/cloud/env/working-with-variables.html).
+
+{:.procedure}
+To add your {{site.data.var.ee}} encryption key to the `env.php` file for each environment:
+
+1. If you have not done so already, use SSH to connect to the Cloud environment.
+
+   ```bash
+   magento-cloud environment:ssh
+   ```
 
-    ```
-    magento-cloud environment:ssh
-    ```
+1. Open the `app/etc/env.php` file in a text editor.
 
-1. Open `app/etc/env.php` in a text editor.
 1. Replace the existing value of `key` with your [{{site.data.var.ee}} key]({{ site.baseurl }}/cloud/setup/first-time-setup-import-prepare.html#encryption-key).
 
-    ```php
-    return array (
-      'crypt' =>
-      array (
-        'key' => '<your encryption key>',
-      ),
-    );
-    ```
+   ```php
```

Comment:
This is a PHP excerpt. Per our style guide "When the PHP code block doesn't have the starting <?php tag, it won't be highlighted appropriately."

```suggestion
   ```php?start_inline=1
```